samples/todolist.md

```diff
@@ -0,0 +1,160 @@
+  - app_name: committime-generic
+    env_from_secrets:
+    - github-secret
+    source_context_dir: exporters/
+    extraEnv:
+    - name: APP_FILE
+      value: committime/app.py
+    - name: LOG_LEVEL
+      value: DEBUG
+    source_ref: master
+    source_url: https://github.com/your_org/pelorus.git
+```
+3. Documentation regarding values.yaml can be found [here](https://github.com/konveyor/pelorus/blob/master/docs/Configuration.md).
+  Apply the updated values for Pelorus by executing:
+```
+helm upgrade pelorus charts/pelorus --namespace pelorus --values /var/tmp/values.yaml
+```
+***Note*** pause to allow the committime exporter pod to build and deploy.
+
+Pelorus should now be installed and configured to measure the todolist sample app. We'll have to deploy the sample application to view measurements from Pelorus's Grafana dashboard.
+
+## Todolist sample application install steps
+
+1. git clone the forked copy of the
+```
+git clone https://github.com/your_org/mig-demo-apps.git
+```
+2. Install the todolist-mongo-go sample application
+```
+cd mig-demo-apps/apps/todolist-mongo-go
+export GITHUB_ORG=<YOUR_REAL_GITHUB_FORK_ORG>
+sed -i 's/your_org/${GITHUB_ORG}/g' mongo-persistent.yaml
+oc create -f mongo-persistent.yaml
+```
+
+The todolist application and mongo database should now build and deploy into the mongo-persistent namespace.
+
+3. Check the build
+```
+oc get all -n mongo-persistent
+```
+***Note*** pause to allow the todolist pod to build and deploy.
+
+## View the Pelorus measurements
+
+1. In your OpenShift Pelorus project page, open the link to granfa or get the link via the cli:
+```
+oc get route -n pelorus | grep grafana
+```
+2. Navigate to *"pelorus / Software Delivery Performance - By App"*
+3. Select the todolist application
+
+You should now see at least one measurement for "Lead time for Change" and "Deployment Frequency"
+
+![gnome-shell-screenshot-3zh4l2](https://user-images.githubusercontent.com/138787/165596818-adc2469e-b3a6-4841-883f-e5018095801f.png)
+
+
+## Optional Github Webhook
+
+You can more easily see how Pelorus works by automatically building and deploying the todolist app if you have configured your source git repository webhooks.
+
+1. To get the build webhook URL you can navigate to the todolist BuildConfig details or via the cli:
+```
+oc get all -n mongo-persistent
+oc describe buildconfig.build.openshift.io/todolist -n mongo-persistent
+```
+***Note*** the secret is hardcoded in the template to be:
+```
+4Xwu0tyAab90aaoasd88qweAasdaqvjknfrl3qwpo
+```
+* Navigate to https://github.com/your_org/mig-demo-apps/settings/hooks
+* Paste the URL with the real secret replacing the text <secret>
+* Toggle SSL as needed, for testing consider disabling.
+* Content type: application/json
+* Click, "Add webhook"
+
+### Test the webhook
+1. Update the git repo and push the change.
+2. A build should trigger in OpenShift
+3. Check the Pelorus committime logs for the commit hash that was pushed
```

Comment:
Somewhat pedantic, but we're checking the exporter output, not the logs. Since the logs are set to debug, inspecting the logs is pretty verbose.